person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  # getter and setter in one
+  attr_accessor :name
+  attr_accessor :age
+
+  # getter
+  attr_reader :id
```

Comment:
- [ ] `:name & :age` can be use on same line, for ex: `attr_accessor :name, :age `.
- [ ] Kindly move the getter & setter before initiaze method, on line 2, Always use getter & setter at top (conventionally)